spec/classes/apache_spec.rb

```diff
@@ -636,7 +636,7 @@
         it "should fail" do
           expect do
             catalogue
-          end.to raise_error(Puppet::Error, /"foo" does not match/)
+          end.to raise_error(Puppet::PreformattedError, /Evaluation Error: Error while evaluating a Resource Statement, Class\[Apache\]: parameter 'sendfile' expects a match for Enum\['Off', 'On', 'off', 'on'\], got 'foo'/)
```

Comment:
is it still needed to test against those cases? In the Vox Pupuli namespace we dropped them